.github/workflows/publish-prod.yaml

```diff
@@ -25,6 +25,8 @@ jobs:
           node-version: 16
       - name: Install Dependencies
         run: npm ci
+      - name: Enable Indexing
+        run: "sed -i 's/noIndex: true/noIndex: false/g' docusaurus.config.js"
```

Comment:
Important to note that the left/right sides of the `sed` command are swapped from what they used to be. Now, in production, we change that line from `noIndex: true` to `noIndex: false`....